dll/ntdll/rtl/libsupp.c

```diff
@@ -642,6 +642,139 @@ RtlPcToFileHeader(IN PVOID PcValue,
     return ImageBase;
 }
 
+/* NOTE: Remove this one once our actctx support becomes better */
+NTSTATUS find_actctx_dll( PUNICODE_STRING pnameW, LPWSTR *fullname, PUNICODE_STRING CallerBuffer, PUNICODE_STRING DynamicString)
```

Comment:
Why is DynamicString a PUNICODE_STRING in this function? it's only checked to be a valid pointer.

Reply:
would a boolean called bAllocate be better?

Reply:
imo, yes :)